examples/src/main/java/com/mapbox/navigation/examples/core/FasterRouteActivity.kt

```diff
@@ -229,6 +235,12 @@ class FasterRouteActivity : AppCompatActivity(), OnMapReadyCallback {
         bottomSheetBehavior = BottomSheetBehavior.from(bottomSheetFasterRoute)
         bottomSheetBehavior.peekHeight = 0
         fasterRouteAcceptProgress.max = MAX_PROGRESS.toInt()
+        navigationMapboxMap?.setOnRouteSelectionChangeListener { route ->
+            mapboxNavigation.setRoutes(mapboxNavigation.getRoutes().toMutableList().apply {
+                remove(route)
+                add(0, route)
+            })
+        }
```

Comment:
why do we have this entire logic 2 times in the file? 🤔 
cc @cafesilencio

Reply:
Oops, that's a mistake. Good catch. I must have gone to lunch between commits.